timesketch/lib/analyzers/contrib/misp_analyzer_test.py

```diff
@@ -70,7 +76,7 @@ def test_attr_match(self, mock_requests_post):
     @mock.patch("requests.post")
     def test_attr_nomatch(self, mock_requests_post):
         """Test no match"""
-        analyzer = misp_analyzer.MispAnalyzer("test_index", 1)
+        analyzer = misp_analyzer.MispAnalyzer("test_index", 1, None, **QUERY_MISP)
         analyzer.misp_url = "https://test.com/"
         analyzer.misp_api_key = "test"
```

Comment:
You have defined the data already in the set-up function above. No need for those two lines.